gradle.properties

```diff
@@ -1,3 +1,5 @@
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 android.enableJetifier=true
 android.useAndroidX=true
+org.gradle.configureondemand=true
```

Comment:
Looks that `configureondemand` is not recommend by Gradle when using Kotlin based Gradle build scripts:
> Enabling the incubating `configuration on demand` feature is not recommended as it can lead to very hard-to-diagnose problems.

https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html#before_you_start_migrating

Reply:
Thanks for finding that - I'll remove this configuration. The configuration time is quite low so this isn't much of a saving anyway (just a few seconds).